graphql/student/mutations.ts

```diff
@@ -246,4 +244,23 @@ export class MutateStudentResolver {
         });
         return getFileURL(file);
     }
+
+    @Mutation(() => Boolean)
+    @Authorized(Role.ADMIN)
+    async studentScheduleCoCReminder(@Arg('studentId') studentId: number) {
```

Comment:
Hmm maybe 'studentRequestCoC' reflects the external meaning better than 'CoC Reminder'?